docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+# this is for development only
+services:
+  db:
+    image: mariadb:10.9.3
+    ports:
+      - 3306:3306
+    volumes:
+      - nosdeputes_data:/var/lib/mysql
+    environment:
+      MARIADB_USER: user
+      MARIADB_PASSWORD: password
+      MARIADB_ALLOW_EMPTY_ROOT_PASSWORD: yes
```

Comment:
je me prends `services.db.environment.MARIADB_ALLOW_EMPTY_ROOT_PASSWORD contains true, which is an invalid type, it should be a string, number, or a null`
ça fonctionne avec juste `MARIADB_ALLOW_EMPTY_ROOT_PASSWORD: 1`

Reply:
fun, j'ai pas ce probleme chez moi mais ok je change :)